vcxagency-node/test/unit/messaging/aconn-msgs.spec.js

```diff
@@ -194,4 +195,64 @@ describe('message download from agent connection', () => {
     expect(msg2).toBeDefined()
     expect(msg2.payload.msg).toBe('annyeong')
   })
+
+  it('should update message status by UID', async () => {
+    // arrange
```

Comment:
There should either the whole arrange - act - assert delineation via comments or none